src/extension/commands/index.ts

```diff
@@ -358,3 +359,8 @@ export async function toggleAutosave(autoSaveIsOn: boolean) {
   }
   return ContextState.addKey(NOTEBOOK_AUTOSAVE_ON, autoSaveIsOn)
 }
+
+export async function runCellWithPrompts() {
+  await ContextState.addKey(NOTEBOOK_RUN_WITH_PROMPTS, true)
```

Comment:
Wait a second, this turn-on prompts for the "session"? It should just do it for a single run of the cell, not be permanent @pastuxso @degrammer.